lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskServiceImpl.java

```diff
@@ -391,6 +391,15 @@ public Task getTask(String id) throws NotAuthorizedOnWorkbasketException, TaskNo
           resultTask.setWorkbasketSummary(workbaskets.get(0));
         }
 
+        TaskSummary resultTaskSummary = createTaskQuery().idIn(id).single();
```

Comment:
This query seems same as the query above. Is there a difference between these two? If not, please change the Exception thrown above instead of adding a new query.